FWIO/RNTuple/plugins/RNTupleSource.cc

```diff
@@ -179,11 +179,16 @@ namespace edm {
     RNTupleInputFile::Options ops;
     ops.enableMetrics_ = enableMetrics_;
     ops.useClusterCache_ = useClusterCache_;
-    file_ = std::make_unique<RNTupleInputFile>(pset.getUntrackedParameter<std::string>("fileName"), ops);
+    auto files = pset.getUntrackedParameter<std::vector<std::string>>("fileNames");
+    if (files.empty()) {
+      throw cms::Exception("NoInputFiles");
+    }
+    file_ = std::make_unique<RNTupleInputFile>(files[0], ops);
```

Comment:
Would it be useful to issue a warning message (if not throw an exception) if the `files.size() > 1`?

Reply:
Up to you. We will eventually support using multiple files.

Reply:
Let's throw an exception then.

Reply:
done